index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
[OPTIONAL] I suggest you add the class you created in your css file to this item.
`<h1 class="display">Hello Microverse</h1>` It makes more sense if the css is being applied to your page.